LSP.sublime-settings

```diff
@@ -19,7 +19,9 @@
   // unless overridden by project-specific settings.
   //
   // Only "source.*" actions are supported.
-  "lsp_code_actions_on_save": {},
+  "lsp_code_actions_on_save": {
+    // "source.fixAll": True
```

Comment:
`True -> true`  
Because you already used the word minor, this would be a patch :)

Reply:
Done